src/data/ACTIONS/MCH.js

```diff
@@ -230,6 +230,7 @@ export default {
 		icon: 'https://xivapi.com/i/003000/003501.png',
 		onGcd: false,
 		cooldown: 6,
+		duration: 20,
```

Comment:
My understanding is that AQ is more in line with DRK's pet, where it just does its own thing independent of the machinist, and is not a damage amp to the caster's GCDs like NIN's Bunshin does, and AQ does not give a buff.  Does it make sense to show it as a buff on the timeline?  If we're going to do that, should we do the same thing with DRK's Living Shadow as well?

Reply:
this question is tricky, I can add functionality to generate such statuses, just for information purposes